### DISCUSSION TITLE
My Unity build was expanding like a boss when I forgot to set the styles of the parent component?

### DISCUSSION THREAD

**Discussion by nyngwang** [jeffreylanters/react-unity-webgl#389] [Help Wanted!]
This is the naive code I use:

```
function UnityBox() {
  return (
    <Page>
      <div className={styles.unitybuild}>
        <div>
          <Unity unityContext={unityContext} />
        </div>
        <div>
          <h1>{title}</h1>
          <p>{intro}</p>
        </div>
      </div>
    </Page>
  )
}
```

<img src="https://user-images.githubusercontent.com/24765272/165779177-fb6196e1-4cca-434f-9b11-96a81033b6a7.jpg" width="600">

(btw, do you know how lag it is that I literally screenshot the screenshot-system itself in the image above?)
(btw, can I call it a "big bang" in my poor Edge browser since the resolution went from `600x300` to `33554432x33554432`. Please let me call this the "milky-way scale" resolution)

I think the plugin should prevent this HUGE size by default. I have to wait about 30 minutes for my: 2.6 GHz 6-Core Intel Core i7, 16 GB 2667 MHz DDR4 memory computer to "just move the cursor".

**Comment by jeffreylanters** [comment]
Hi. This is a know issue caused by Unity, simply providing a height and width to the canvas will resolve this problem.

**Reply by nyngwang** [reply]
OK, thanks for the answer.
